src/main/java/exception/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package exception;
+
+public class DukeException extends Exception {
+    public DukeException(String issue) {
```

Comment:
Maybe can add more logic in your DukeException to further classify your DukeException and its messages. That way you can reuse the same messages if need be.